handler/src/test/java/io/netty5/handler/ssl/SslUtilsTest.java

```diff
@@ -17,45 +17,42 @@
 
 import io.netty.buffer.ByteBuf;
 import io.netty.buffer.Unpooled;
+import io.netty5.buffer.api.Buffer;
 import org.junit.jupiter.api.Test;
 
 import javax.net.ssl.SSLContext;
 import javax.net.ssl.SSLEngine;
 import javax.net.ssl.SSLException;
 import java.nio.ByteBuffer;
-import java.nio.ByteOrder;
 import java.security.NoSuchAlgorithmException;
 
+import static io.netty5.buffer.api.DefaultBufferAllocators.offHeapAllocator;
 import static io.netty5.handler.ssl.SslUtils.getEncryptedPacketLength;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 public class SslUtilsTest {
 
-    @SuppressWarnings("deprecation")
     @Test
     public void testPacketLength() throws Exception {
-        SSLEngine engineLE = newEngine();
         SSLEngine engineBE = newEngine();
 
         ByteBuffer empty = ByteBuffer.allocate(0);
-        ByteBuffer cTOsLE = ByteBuffer.allocate(17 * 1024).order(ByteOrder.LITTLE_ENDIAN);
         ByteBuffer cTOsBE = ByteBuffer.allocate(17 * 1024);
 
-        assertTrue(engineLE.wrap(empty, cTOsLE).bytesProduced() > 0);
-        cTOsLE.flip();
-
         assertTrue(engineBE.wrap(empty, cTOsBE).bytesProduced() > 0);
         cTOsBE.flip();
 
-        ByteBuf bufferLE = Unpooled.buffer().order(ByteOrder.LITTLE_ENDIAN).writeBytes(cTOsLE);
-        ByteBuf bufferBE = Unpooled.buffer().writeBytes(cTOsBE);
-
-        // Test that the packet-length for BE and LE is the same
-        assertEquals(getEncryptedPacketLength(bufferBE, 0), getEncryptedPacketLength(bufferLE, 0));
-        assertEquals(getEncryptedPacketLength(new ByteBuffer[] { bufferBE.nioBuffer() }, 0),
-                getEncryptedPacketLength(new ByteBuffer[] { bufferLE.nioBuffer().order(ByteOrder.LITTLE_ENDIAN) }, 0));
+        try (Buffer buffer = offHeapAllocator().copyOf(cTOsBE);
+             Buffer copy = buffer.copy()) {
+            buffer.forEachReadable(0, (index, component) -> {
+                ByteBuffer[] byteBuffers = { component.readableBuffer() };
+                assertEquals(getEncryptedPacketLength(byteBuffers, 0),
+                             getEncryptedPacketLength(copy, 0));
+                return false;
```

Comment:
why false ?

Reply:
There should only be one component. We can make that an assert, though, so it doesn't sneak past us if it ever changes.